src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -374,10 +374,10 @@ public String getQuestionAdditionalInfoHtml(int questionNumber, String additiona
 
     @Override
     public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> responses,
-            FeedbackQuestionAttributes question,
-            String studentEmail,
-            FeedbackSessionResultsBundle bundle,
-            String view) {
+                                                  FeedbackQuestionAttributes question,
```

Comment:
Same here. Pls refer to https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html